apps/src/templates/HourOfCodeGuideEmailDialog.jsx

```diff
@@ -9,10 +9,15 @@ import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
 import {Heading1, Heading3} from '@cdo/apps/componentLibrary/typography';
 import experiments from '@cdo/apps/util/experiments';
 import style from './hoc-guide-dialogue.module.scss';
+import {isEmail} from '@cdo/apps/util/formatValidation';
 
-function HourOfCodeGuideEmailDialog({isSignedIn}) {
+function HourOfCodeGuideEmailDialog({isSignedIn, unitId}) {
```

Comment:
With the switch to using unit id instead of course offering id, we'll need to update the email too since that is expecting a course offering (not in the scope of this PR for sure, but just wanted to make a note)!

Reply:
Thanks for flagging! I'm curious- how does the email use that field?